P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1326 @@
+          </post>
+        </egXML>
+      </p>
+      <p>The <att>replyTo</att> attribute is used to capture information drawn from the original
+        metadata associated with a post that asserts to which previous post the current post is a
+        response, or to which previous post it refers. This metadata is included by many, but not
+        all, CMC environments, when the user executes a formal reply action (e.g., by clicking or
+        tapping a reply button). This attribute should not be used to encode interpreted or inferred
+        reply relations based on linguistic cues or discourse markers.</p>
+      <p>The <att>replyTo</att> attribute indicates the replied-to or referred-to posts by providing
+        one or more pointers to them. In the following example, reply references in the source
+        indicate that the first <gi>post</gi> is a reply to an initial post that is not part of the
+        example, the second is a reply to the first, and the third is a reply to the second.
+        <egXML xml:id="e9" xmlns="http://www.tei-c.org/ns/Examples" xml:lang="de" source="#BIB_scilog1">
+          <post type="comment" modality="written" generatedBy="human" xml:id="cmc_post10" who="#u7"
+            replyTo="#cmc_ex_p9" when-iso="2015-07-29T21:44">
+            <p>Es hat den Anschein, als wäre bei BER durchaus große Kompetenz am Bau, allerdings
+              nicht in Form von Handwerkern….</p>
+            <p>http://www.zeit.de/2015/29/imtech-flughafen-berlin-ber-verzoegerung/komplettansicht</p>
+          </post>
+          <post type="comment" modality="written" generatedBy="human" xml:id="cmc_post11" who="#u8"
+            replyTo="#cmc_ex_p10" when-iso="2015-07-30T19:11">
+            <p>Nein Nein, an den Handwerkern kann es rein strukturel nicht gelegen haben. Niemand
+              lässt seine Handwerker auf der Baustelle derart allein. Zudem gibt es höchstoffizielle
+              “Abnahmen” von Bauabschnitten/phasen. Welcher Mangel auch bestanden hatte, er hätte
+              Zeitnah auffallen müssen.</p>
+            <p>Uuups, für Imtek hab ich mal in einer Nachunternehmerfirma gearbeitet. Imtek is
+              offenbar ein universeler Bauträger, der alles baut.</p>
+          </post>
+          <post type="comment" modality="written" generatedBy="human" xml:id="cmc_post12" who="#u8"
+            replyTo="#cmc_ex_p11" when-iso="2015-07-30T19:26">
+            <p>Stahlkunstruktionen dacht ich mal, was die bauen—oder bauen lassen.</p>
+            <p>Das ist schon ein übles Ding. Die Ausschreibungenund Angebote sind unauffällig, aber
+              wenn Unregelmässigkeiten auftreten (im Bauverlauf) dann gibt es die saftigen
+              Rechnungen. Da steht dann der Bauherr da und fragt sich, wie er denn so schnell einen
+              fähigen Ersatz herbekommt. Und diese Frage erübrigt sich meist, weil der Markt der
+              Baufirmen das nicht hergibt—weil tendenziel 100 % Auslastung. (und noch schlimmer:
```

Comment:
hergibt—weil -> there should be a space before and after the — (also checked in the original).